packages/create-wmr/tpl/package.json

```diff
@@ -12,10 +12,10 @@
 		"react-dom": "preact/compat"
 	},
 	"dependencies": {
-		"preact": "^10.5.12",
-		"preact-iso": "^0.2.0"
+		"preact": "latest",
```

Comment:
imo specifying `latest` as a default for users isn't the best thing, certainly not for newcomers, my suggestion would be to get the current latest and prefix with `^` ofcourse, that could be for another PR

Reply:
Was just writing something similar.

`create-wmr`, at least at the moment, does install `latest` on project creation. So we could instead specify versions there or `latest` here, just something to keep them matched up.

Reply:
I ended up just versioning in `pkg-install` & matched that in the template's `package.json`. 

`preact-iso` (for now) gets `0.x` as I'm guessing that's better than locking them to a minor with the 0-major semver rules.